crates/turborepo-lib/src/cli.rs

```diff
@@ -204,11 +204,10 @@ impl Args {
     pub fn new() -> Result<Self> {
         // We always pass --single-package in from the shim.
         // We need to omit it, and then add it in for run.
-        let arg_separator_position =
-            std::env::args_os().position(|input_token| input_token == "--");
+        let arg_separator_position = env::args_os().position(|input_token| input_token == "--");
 
         let single_package_position =
-            std::env::args_os().position(|input_token| input_token == "--single-package");
+            env::args_os().position(|input_token| input_token == "--single-package");
```

Comment:
Why are these changes in this PR? If this is just lint fixes, can you make them separately?

Reply:
Yeah this was just to test the CI. I'll revert them out